oauth2client/client.py

```diff
@@ -811,13 +811,13 @@ def _do_refresh_request(self, http_request):
       raise AccessTokenRefreshError(error_msg)
 
   def _revoke(self, http_request):
-    """Revokes the refresh_token and deletes the store if available.
+    """Revokes this token and deletes the stored copy (if available).
```

Comment:
Is "this token" the right wording? Why not "these Oauth2Credentials"? You are referring to "this object", right, and the object is an Oauth2Credentials?